kratos/python/add_strategies_to_python.cpp

```diff
@@ -12,9 +12,8 @@
 
 // System includes
 
-
 // External includes
-#ifdef KRATOS_USE_AMATRIX   // This macro definition is for the migration period and to be removed afterward please do not use it
+#ifdef KRATOS_USE_AMATRIX                 // This macro definition is for the migration period and to be removed afterward please do not use it
```

Comment:
The file changes are confusing. What happened here?

Reply:
wrong formatting, removed